extension/src/experiments/model/status/collect.ts

```diff
@@ -22,11 +46,11 @@ const collectStatus = (
   unassignColors?: Color[]
 ) => {
   const { id, status } = experiment
-  if (!id || isQueued(status) || hasKey(acc, id)) {
+  if (!id || isQueued(status) || (id !== 'workspace' && hasKey(acc, id))) {
```

Comment:
careful with this, need to make sure that workspace can be unselected and not magically selected again